test/images/cloudbuild.yaml

```diff
@@ -33,4 +33,4 @@ substitutions:
   # a branch like 'master' or 'release-0.2', or a tag like 'v0.2'.
   _PULL_BASE_REF: 'master'
   # _WHAT will contain the image name to be built and published to the staging registry.
-  _WHAT: 'all-conformance'
+  _WHAT: 'updated-images'
```

Comment:
The job will run only on the modified images IIUIC
https://github.com/kubernetes/test-infra/blob/b6a3b30009d28054eda88515dc9c0f25e94d343e/config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml#L6